jolly-jellyfish/src/static/style.css

```diff
@@ -0,0 +1,102 @@
+/**
+ * A global style sheet.
+ */
+
+@import url('https://fonts.googleapis.com/css2?family=News+Cycle&display=swap');
+
+@font-face {
+  font-family: 'hellovetica';
+  src: url('/static/fonts/hellovetica.woff2') format('woff2');
+  font-weight: normal;
+  font-style: normal;
+  unicode-range: U+21,U+22,U+27,U+2C-2E,U+3A,U+3B,U+3F,U+41-5A,U+61-7A;
+}
+
+html {
+    min-height: 100%;
+}
+
+body {
+    background: #008081;
+    font-family: hellovetica, "MS Sans Serif", "News Cycle", sans-serif;
+    font-size: 10px;
+    min-height: 100%;
+    background: #008081 url("/static/images/win95-toolbar.png") center bottom / 100% auto no-repeat;
+}
+
+main {
+    background: #c0c0c0;
+    max-width: 890px;
+    margin: 50px 0 60px 30px;
+    padding: 10px;
+    box-shadow: -1px -1px 2px #000 inset, 1px 1px 2px #fff inset;
+}
+
+input[type="text"], input[type="password"], input[type="email"], textarea {
+    background: white;
+    border-top: 2px solid black;
+    border-left: 2px solid black;
+    border-right: 2px solid white;
+    border-bottom: 2px solid white;
+    padding: 3px;
+    font-family: hellovetica, "MS Sans Serif", "News Cycle", sans-serif;
```

Comment:
Shouldn't this inherit from `body`?

Reply:
No, the contents of input fields don't inherit fonts.